examples/prefab/resources/display_config.ron

```diff
@@ -1,10 +1,4 @@
 (
-  dimensions: None,
-  max_dimensions: None,
-  min_dimensions: None,
-  fullscreen: false,
   multisampling: 0,
   title: "Prefab example",
-  visibility: true,
-  vsync: true,
```

Comment:
Why are we modifying this display config?

Reply:
To demonstrate how to use it so that when people want to know where the features are they can find them.

Reply:
There are 15 other examples with full config, I don't see the point of having it in full mode in all of them.

Reply:
Ok, that's fine with me then